include/wx/msw/private/dpiaware.h

```diff
@@ -0,0 +1,72 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/msw/private/dpiaware.h
+// Purpose:     AutoSystemDpiAware class
+// Author:      Maarten Bent
+// Created:     10/6/2016
+// Copyright:   (c) Maarten Bent
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_MSW_DPI_AWARE_H_
+#define _WX_MSW_DPI_AWARE_H_
+
+#ifndef WX_PRECOMP
+    #include "wx/msw/missing.h"
+#endif
+
+#include "wx/dynlib.h"
+
+#if wxUSE_DYNLIB_CLASS
+
+// ----------------------------------------------------------------------------
+// Temporarily change the DPI Awareness context to System
+// ----------------------------------------------------------------------------
+
+class AutoSystemDpiAware
+{
+    #define WXDPI_AWARENESS_CONTEXT_UNAWARE      ((WXDPI_AWARENESS_CONTEXT)-1)
+    #define WXDPI_AWARENESS_CONTEXT_SYSTEM_AWARE ((WXDPI_AWARENESS_CONTEXT)-2)
+    typedef WXDPI_AWARENESS_CONTEXT
```

Comment:
I'm a bit confused: where is `WXDPI_AWARENESS_CONTEXT` defined? I can't find it anywhere...

Reply:
https://github.com/wxWidgets/wxWidgets/blob/master/include/wx/defs.h#L2833